src/main/java/io/ebean/bean/EntityBeanIntercept.java

```diff
@@ -1196,4 +1198,18 @@ public void setLoadError(int propertyIndex, Exception t) {
     }
     return ret;
   }
+
+  /**
+   * Returns if the entity is part of a BeanCollection. This can be used to track movement from one collection to another.
```

Comment:
```suggestion
   * Returns true if the entity was removed from a BeanCollection. This can be used to track movement from one collection to another.
```